webrtc.html

```diff
@@ -2087,19 +2126,7 @@
               </li>
 
               <li>
-                <p>If <var>connection</var>'s <a href=
-                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
-                signalingState</a> is <code>stable</code>, then fire an <a href=
-                "#event-negotiation">negotiationneeded</a> event at
-                <var>connection</var>.</p>
-
-                <div class="note">
-                  <p>Throughout this document, we need to indicate what should
-                  happen if a change happens and the state is not stable, most
-                  likely to fire the negotiationneeded event when the state
-                  eventually returns to stable.
```

Comment:
I think that an internal property is OK.  Applications that require more complex behaviour will likely understand the complexities well enough to maintain their own understanding of what the variable's value is; the value is largely there for applications that don't care to track state, so they are unlikely to do anything other than observe the event.